torch/csrc/jit/runtime/serialized_shape_function_registry.cpp

```diff
@@ -16,7 +16,7 @@
 namespace torch::jit {
 
 
-std::string shape_funcs = ""
+static std::string shape_funcs = ""
```

Comment:
Is this constexpr able? Might need to wait until C++20 until it is though..

Reply:
JIT code is not actively developed, so I prefer not to change it.